tests/bud.bats

```diff
@@ -15,12 +15,14 @@ load helpers
 @test "bud: build manifest list and --add-compression zstd" {
   start_registry
   run_buildah login --tls-verify=false --authfile ${TEST_SCRATCH_DIR}/test.auth --username testuser --password testpassword localhost:${REGISTRY_PORT}
-  run_buildah build $WITH_POLICY_JSON -t image1 --platform linux/amd64 -f $BUDFILES/dockerfile/Dockerfile
-  run_buildah build $WITH_POLICY_JSON -t image2 --platform linux/arm64 -f $BUDFILES/dockerfile/Dockerfile
+
+  imgname="img$(random_string | tr A-Z a-z)"
```

Comment:
q: would the safename helper from podman make sense for the parallel work here or do we not need it because the tests do not share storage?

Reply:
I almost went with `safename` but couldn't think of any real-world situation in which it would help: the errors I've seen in testing only show layer IDs. It's not much work to bring it in. Should I?

Reply:
The one thing would be that we can skip the ` | tr A-Z a-z` that is duplicated in so many places. But that could also be fixed by random string only returning lowercase. I am not sure what would be better.
Either way not a blocker.